packages/runtime/src/shared/types.ts

```diff
@@ -42,19 +42,20 @@ export interface FlexboxStyles {
   maxWidth?: string;
   minWidth?: string;
   visible?: boolean;
+  childrenFits?: IsFitted extends true ? string[] : never;
```

Comment:
Just compose the types together - what does this to do that FlexboxStyles & IsFitted does not? Keep it simple.